proc/process.c

```diff
@@ -757,6 +781,51 @@ int process_load(process_t *process, vm_object_t *o, offs_t base, size_t size, v
 	if (process_relocate(reloc, relocsz, (char **)entry) < 0)
 		return -ENOEXEC;
 
+#ifdef __sparc__
```

Comment:
Something for later, but we need to aspire to remove this `ifdef`. We could make a distinction by presence of `.rela` header